docs/GDI-SOP_charter.md

```diff
@@ -0,0 +1,167 @@
+
+For example, if a GDI node already functions as an archive with submitted datasets, the curation of these datasets, when outside the GDI scope, is not expected to follow a hypothetical "Dataset Curation SOP" from GDI.
+
+Therefore, it is the **responsibility of the GDI nodes to implement these SOPs when the use case pertains to GDI infrastructure**, governance, or data.
+
+The scope of these SOPs is focused on roles, topics, and resources that are essential for establishing or operating node services, ensuring high-quality data management, and maintaining robust technical infrastructure. However, certain items are deliberately excluded from the current scope to maintain focus.
+
+### In-Scope
+#### Roles
+- **Helpdesk Staff and Operations Bioinformaticians**: Responsible for responding to and resolving user queries and issues within the node.
+- **Data Stewards, Data Curators, and Bioinformaticians**: Focused on the preparation, harmonization, and curation of data and metadata to ensure high-quality data management.
+- **Software Developers, Software Engineers, and DevOps Engineers**: Tasked with designing, developing, implementing, deploying, and monitoring software and services deployed at a node.
+
+#### Topics
+- **Data Protection & Security**: Includes procedures related to security incident response, risk assessment, and compliance with data protection regulations (e.g., DPA, DAA, DPIA).
+- **Data & Metadata Management**: Covers data management plans (DMPs), and processes for ensuring data quality and harmonization within the node, including interactions with the GDI User Portal.
+- **Technical Infrastructure & Software Development**: Focuses on technical certifications, compliance assessment tools, and ensuring that node infrastructure meets GDI standards.
+- **Helpdesk & Operations**: Encompasses standard operating procedures for helpdesk interactions with users, including interfacing with the GDI User Portal.
+
+#### Resources
+- **SOP Examples & Templates**: Provision of standardized templates for SOPs that can be adapted by nodes.
+- **Operational Recommendations & Guidelines**: Guidance on best practices for operating node services within the GDI network.
+- **Training on SOP Usage & Management**: Training materials and sessions to ensure that node staff can effectively implement and manage SOPs.
+- **Resource Management Plan**: A plan for the management of resources required for SOP implementation.
+
+### Out-of-scope
+#### Roles
+- **Researchers Using Node Services**: While crucial to the broader GDI network, their specific operational procedures are not covered by these SOPs.
```

Comment:
The user base of GDI is much broader, so maybe we should include some more roles in the list. This is what I can read on the website, https://gdi.onemilliongenomes.eu/:

> Controlled access will be given to approved clinicians, scientists in the public and private sector and healthcare policy makers.

In future, I guess that data subjects also may be using our services